docs/samples/pvc/README.md

```diff
@@ -0,0 +1,31 @@
+
+# Create a InferenceService for on-prem cluster
+
+The guide shows how to train model and create InferenceService for the trained model for on-prem cluster.
+
+## Prerequisites
+Refer to the [document](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) to create Persistent Volume (PV) and Persistent Volume Claim (PVC), the PVC will be used to store model.
```

Comment:
Add a sample YAML that creates a local PVC and a sample command to upload the model to the PVC?

Reply:
@rakelkar Thanks. I think no need to add yaml for PVC since that's basic skill for k8s and there are some types of PVC we cannot sure user will use which one.
The next step "Trainning model" will generate a model to the PVC.